src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -446,6 +446,17 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: SingleArgTrig(x => x * 180.0 / Math.PI))
             },
+            {
+                BuiltinFunctionsCore.Dec2Hex,
+                StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.Dec2Hex.Name,
+                    expandArguments: InsertDefaultValues(outputArgsCount: 2, fillWith: new BlankValue(IRContext.NotInSource(FormulaType.Blank))),
```

Comment:
>new BlankValue(IRContext.NotInSource(FormulaType.Blank)) [](http://example.com/codeflow?start=87&length=56)

`new NumberValue(IRContext.NotInSource(FormulaType.Number), 0)` - `0` is the default value for the second argument